packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/close/proposeConfig.ts

```diff
@@ -0,0 +1,68 @@
+import { RDD } from '@chainlink/gauntlet-terra'
+import { CATEGORIES } from '../../../../lib/constants'
+import { AbstractInstruction, BeforeExecute, instructionToCommand } from '../../../abstract/executionWrapper'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { ContractInput } from '../proposeConfig'
+import { MnemonicKey } from '@terra-money/terra.js'
+
+type CommandInput = {
+  proposalId: number
+}
+
+const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInput> => {
+  if (flags.input) return flags.input as CommandInput
+  return {
+    proposalId: flags.configProposal,
+  }
+}
+
+const makeContractInput = async (input: CommandInput): Promise<ContractInput> => {
+  const hexToBase64 = (s) => Buffer.from(s, 'hex').toString('base64')
```

Comment:
I've seen an fn like this defined in-line in multiple places. Consider extracting it somewhere.